test/karma/helpers/ParentClickView.js

```diff
@@ -0,0 +1,35 @@
+var _ = require('underscore');
+var TorsoView = require('./../../../modules/View');
+var ChildClickView = require('./ChildClickView');
+
+module.exports = TorsoView.extend({
+  events: {
+    'click div.parent' : 'myClick'
+  },
+  initialize: function() {
+    this.spyHook();
+
+    spyOn(this, 'myClick');
+    spyOn(this, 'afterMyEvent');
+    spyOn(this, 'afterMyDeactivatableEvent');
```

Comment:
okay, i did some thinking about spyOn's. I don't like the testing framework to be in our source objects. Source files, sure, but someone should be able to look at the Torso View or Model and not see any manipulation for the testing framework. This is because we don't want them confused about what's necessary for the functionality vs the testing.

So, if we care about this, manipulating the initialize method whether its to setup spys or a hook for setting up spys is not ideal.

We have two options to handle this:
1. every time we instantiate the view in the test, we add the spy on the initialized view. Most cases, we don't apply any spies because we don't use them very often. I'm acutally okay with this if you think this is the best option.
2. Use the backboneSpy.js that was already created for this exact reason, that adds a prototype-level wrapper that finds the instance that was invoking it and adds the spy. All jsdom tests use the spyOnBackbone thing, which makes spies work on the prototype.